pkg/cfn/manager/tasks.go

```diff
@@ -87,7 +100,10 @@ func (c *StackCollection) CreateAllNodeGroups(onlySubset sets.String) []error {
 		}
 		t := Task{
 			Call: c.CreateNodeGroup,
-			Data: ng,
+			Data: DataWithPrinter{
```

Comment:
I'm actually reworking how this is done, we are moving away from `interface{}` and adding a few other things, I'm due about to open a PR sometime today :)

Reply:
nice!